arm-kobo-linux-gnueabihf.cmake

```diff
@@ -9,8 +9,8 @@ set(GCC_COMPILER_VERSION "" CACHE STRING "GCC Compiler version")
 set(GNU_MACHINE "arm-kobo-linux-gnueabihf" CACHE STRING "GNU compiler triple")
 set(ENABLE_NEON TRUE)
 
-set(TARGET_SYSROOT /home/$ENV{USER}/kobo/x-tools/arm-kobo-linux-gnueabihf/arm-kobo-linux-gnueabihf/sysroot/)
```

Comment:
Not sure what this kobo folder is doing here. It's present in some scripts and absent in some....